packages/vanilla/src/components/footer/footer.template.js

```diff
@@ -0,0 +1,87 @@
+import footerNav from './footerNav.template';
+import { Accordion } from 'carbon-components';
+import { settings } from 'carbon-components';
+
+const { prefix } = settings;
+
+/**
+ * renders accordion components for footer on smaller screen sizes
+ *
+ * @param {string} El type of footer in use
+ * @returns {object} JSX object
+ */
+function footerCreate(El) {
+  const accordionEl = El.querySelector('.bx--accordion');
+  if (accordionEl) {
+    // eslint-disable-next-line
+    const accordionInit = Accordion.create(accordionEl);
+  }
+}
+/**
+ * renders logo and bottom section of footer
+ *
+ * @param {string} type type of footer in use
+ * @returns {object} JSX object
+ */
+const footerTemplate = ({ type }) => {
+  return `
+  <footer data-autoid="dds--footer" class="bx--footer ${setFooterType(type)}">
```

Comment:
@andysherman2121 can you pull in prefix for all of the `bx--` classnames here, as well as `dds--` for the stableselector prefix